runtime/src/bank.rs

```diff
@@ -375,33 +375,38 @@ impl CachedExecutors {
         })
     }
 
-    fn put(&mut self, pubkey: &Pubkey, executor: Arc<dyn Executor>) {
-        let entry = if let Some(mut entry) = self.executors.remove(pubkey) {
-            entry.executor = executor;
-            entry
-        } else {
-            if self.executors.len() >= self.max {
-                let mut least = u64::MAX;
-                let default_key = Pubkey::default();
-                let mut least_key = &default_key;
-
-                for (key, entry) in self.executors.iter() {
-                    let count = entry.prev_epoch_count + entry.epoch_count.load(Relaxed);
-                    if count < least {
-                        least = count;
-                        least_key = key;
-                    }
-                }
-                let least_key = *least_key;
-                let _ = self.executors.remove(&least_key);
+    fn put(&mut self, pubkey: Pubkey, executor: Arc<dyn Executor>) {
+        match self.executors.entry(pubkey) {
+            Entry::Vacant(entry) => {
+                entry.insert(CachedExecutorsEntry {
+                    prev_epoch_count: u64::default(),
```

Comment:
Just curious, why use `::default()` here instead of 0?

Reply:
Mostly just out of habit; It is bit indicative that I am initializing all these fields with their default values.
But I guess for u64 it does not really matter either way.